.travis.yml

```diff
@@ -36,6 +36,10 @@ matrix:
       env: DJANGO="django>=1.10.0,<1.11.0" REST_FRAMEWORK="djangorestframework>=3.4,<3.5"
     - python: "3.3"
       env: DJANGO="django>=1.9.0,<1.10.0" REST_FRAMEWORK="djangorestframework>=3.4,<3.5"
+  include:
+    - python: "3.5"
+      env: W='python -W error -m' DJANGO='https://github.com/django/django/archive/master.tar.gz' REST_FRAMEWORK='djangorestframework>=3.4,<3.5'
   allow_failures:
+    - env: W='python -W error -m' DJANGO='https://github.com/django/django/archive/master.tar.gz' REST_FRAMEWORK='djangorestframework>=3.4,<3.5'
```

Comment:
The intent here is to have a failing build in the Travis output — to give us something to look at — without needing to routinely read the build output, which we won't do. 

Thus, perhaps how it was for now?